setup.py

```diff
@@ -135,71 +150,73 @@ def readme():
 #)
 
 ext_quantile = Extension(name="quantile_alg",
-                             libraries = MPI_LIBS,
-                             sources=["hpat/_quantile_alg.cpp"],
-                             include_dirs=[PREFIX_DIR+'/include/'],
-                             extra_compile_args=['-std=c++11'],
-                             extra_link_args=['-std=c++11'],
-                             )
+                         sources=["hpat/_quantile_alg.cpp"],
+                         libraries = MPI_LIBS,
+                         extra_compile_args = eca,
+                         extra_link_args = ela,
+                         include_dirs = ind,
+                         library_dirs = lid,
+)
 
-pq_libs = MPI_LIBS + ['boost_filesystem']
 
-if is_win:
-    pq_libs += ['arrow', 'parquet']
-else:
-    # seperate parquet reader used due to ABI incompatibility of arrow
-    pq_libs += ['hpat_parquet_reader']
```

Comment:
If we remove hpat_parquet_reader as a separate package, we can't build HPAT with default Anaconda environment anymore. Can we have both default Anaconda and Intel builds? maybe if macros?

Reply:
What happens with anaconda?

Reply:
Anaconda is not compatible with conda-forge (we use parquet/arrow) because of compiler version differences (gcc 7.2 vs 4.8). hpat_parquet_reader provides a C interface (without std::string) to avoid the issue.

Reply:
I guess the bigger problem is boost, which we use from conda-forge.

Reply:
It's easier to go around boost problems.